mozilla/balrog
# patch 1380874: Remove signoff text from history

#### What does this PR do?
Removes text about ` signoffs` from history pages.
#### How should this be manually tested?
Once `balrog` is running on `http://127.0.0.1:8080` navigate to `Rules` and schedule a change.
Click on the `history` button on that scheduled change, there should be no text about `signoffs`.
#### What are the relevant bugzilla stories?
https://bugzilla.mozilla.org/show_bug.cgi?id=1380874
